plugins/modules/sonic_vlans.py

```diff
@@ -111,6 +111,64 @@
 #sonic#
 #
 
+# Using replaced
+
+# Before state:
+# -------------
+#
+#sonic# show Vlan
+#Q: A - Access (Untagged), T - Tagged
+#NUM        Status      Q Ports
+#10         Inactive
+#30         Inactive
+#
+#sonic#
+
+- name: Replace all attributes of specified VLANs with provided configuration
+  dellemc.enterprise_sonic.sonic_vlans:
+    config:
+      - vlan_id: 10
+    state: replaced
+
+# After state:
+# ------------
+#
+#sonic# show Vlan
+#Q: A - Access (Untagged), T - Tagged
+#NUM        Status      Q Ports
+#10         Inactive
+#30         Inactive
+#
+#sonic#
+
+# Using overridden
+
+# Before state:
+# -------------
+#
+#sonic# show Vlan
+#Q: A - Access (Untagged), T - Tagged
+#NUM        Status      Q Ports
+#10         Inactive
+#30         Inactive
+#
+#sonic#
+
+- name: Override device configuration of all VLANs with provided configuration
+  sonic_vlans:
```

Comment:
```suggestion
  dellemc.enterprise_sonic.sonic_vlans:
```
Please update the module name as in rest of the examples.

Reply:
"dellemc.enterprise_sonic." added to module name.

Reply:
Corrected. Thanks!